libs/wire-subsystems/src/Wire/UserGroupStore/Postgres.hs

```diff
@@ -220,6 +221,94 @@ getUserGroup team id_ includeChannels = do
             where ug.id = ($1 :: uuid) and ug.team_id = ($2 :: uuid)
           |]
 
+getUserGroupsWithMembers ::
+  forall r.
+  ( UserGroupStorePostgresEffectConstraints r
+  ) =>
+  UserGroupPageRequest ->
+  Sem r UserGroupPageWithMembers
+getUserGroupsWithMembers req = runTransaction TxSessions.ReadCommitted TxSessions.Read $ do
+  groups :: [UserGroup] <- Tx.statement () $ refineResult (mapM toUserGroup) $ buildStatement query rows
+  count <- getUserGroupCount req
+  return $ UserGroupPage groups count
```

Comment:
The convention is to use `pure`. I wouldn't care so much if my linter would mark this red... :)